File: java/Bioskop.java
class Film {
    private int id;
    private String poster;
    private String judul;
    private String genre;
    private int durasi;
    private int harga;
    private String sutradara;


    // constructor
    public Film(int id, String poster, String judul, String genre, int durasi, int harga, String sutradara) {
        this.id = id;
        this.poster = poster;
        this.judul = judul;
        this.genre = genre;
        this.durasi = durasi;
        this.harga = harga;
        this.sutradara = sutradara;
    }

    // ---- Getter ----
    public int getId() { return id; }
    public String getPoster() { return poster; }
    public String getJudul() { return judul; }
    public String getGenre() { return genre; }
    public int getDurasi() { return durasi; }
    public int getHarga() { return harga; }
    public String getSutradara() { return sutradara; }

    // ---- Setter ----
    public void setId(int id) { this.id = id; }
    public void setPoster(String poster) { this.poster = poster; }
    public void setJudul(String judul) { this.judul = judul; }
    public void setGenre(String genre) { this.genre = genre; }
    public void setDurasi(int durasi) { this.durasi = durasi; }
    public void setHarga(int harga) { this.harga = harga; }
    public void setSutradara(String sutradara) { this.sutradara = sutradara; }
}

File: cpp/Bioskop.cpp
#include <string>
using namespace std;

class Film {
private:
    int id;
    string poster;
    string judul;
    string genre;
    int durasi;
    int harga;
    string sutradara;

public:
    // Constructor koosng
    Film();

    // constructor berparameter
    
    Film(int id, string poster, string judul, string genre, int durasi, int harga, string sutradara) {
        this->id = id;
        this->poster = poster;
        this->judul = judul;
        this->genre = genre;
        this->durasi = durasi;
        this->harga = harga;
        this->sutradara = sutradara;
    }

    // ---- Getter ----
    int getId() { return id; }
    string getPoster() { return poster; }
    string getJudul() { return judul; }
    string getGenre() { return genre; }
    int getDurasi() { return durasi; }
    int getHarga() { return harga; }
    string getSutradara() { return sutradara; }

    // ---- Setter ----
    void setId(int id) { this->id = id; }
    void setPoster(string poster) { this->poster = poster; }
    void setJudul(string judul) { this->judul = judul; }
    void setGenre(string genre) { this->genre = genre; }
    void setDurasi(int durasi) { this->durasi = durasi; }
    void setHarga(int harga) { this->harga = harga; }
    void setSutradara(string sutradara) { this->sutradara = sutradara; }
};

